### PR TITLE
descriptor.md:fix info

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

File: descriptor.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the media type of the referenced content.
+  This OPTIONAL property contains the media type of the referenced content.
```
Comment (wking) [inline descriptor.md]: This should be REQUIRED.  Without it, a consumer will not know how to interpret the referenced content.

**Comment by stevvooe** [comment]
Closing because this is wrong. MediaType is required on descriptors.

@q384566678 I am really having trouble making sense of your PRs. Could you please take the time to link to an issue or describe why you are proposing a change?

**Comment by q384566678** [comment]
@stevvooe I made this changes because of [this](https://github.com/opencontainers/image-spec/blame/master/specs-go/v1/descriptor.go#L23).Perhaps I do not have a good understanding of this problem.

**Comment by stevvooe** [comment]
@q384566678 What do you mean by that? That is just a link to a line of code.

**Comment by wking** [comment]
> What do you mean by that? That is just a link to a line of code.

Maybe that having `omitempty` for a required property is strange.  I suggest dropping `omitempty` (and keeping descriptor `mediaType` required).

**Comment by q384566678** [comment]
> Maybe that having omitempty for a required property is strange. I suggest dropping omitempty (and keeping descriptor mediaType required).

Agree,
Before I look at this code, `omitempty` let me mistakenly believe that it is an optional field, so it caused the wrong PR.

**Comment by wking** [comment]
> … `omitempty` let me mistakenly believe that it is an optional field…

I've filed #553 removing it.

**Comment by stevvooe** [comment]
> Maybe that having omitempty for a required property is strange. I suggest dropping omitempty (and keeping descriptor mediaType required).

This is terrible practice. `omitempty` and a field requirement are not related.

**Pull request closed without merging**